package.json

```diff
@@ -60,16 +61,19 @@
     "audit:public": "npm audit --registry https://registry.npmjs.org/"
   },
   "dependencies": {
-    "@brightside/imperative": ">=2.0.0-0",
+    "@brightside/imperative": "^2.1.0",
+    "@types/ssh2": "^0.5.38",
     "js-yaml": "3.9.0",
+    "popper.js": "^1.14.7",
+    "ssh2": "^0.8.2",
```

Comment:
@BrandonJenkins14   The TPSR table in the zowe doc might need to be updated with these new dependencies